src/Parser/FullParseResult.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace RecursiveTree\Seat\TreeLib\Parser;
+
+use Illuminate\Support\Collection;
+
+class FullParseResult extends ParseResult
```

Comment:
I think it would make more sense to merge this into the ParseResult class and just pass an empty list for parsers that don't support it yet. I can add support for them myself.

Reply:
Okay I agree, I'll update the PR with a new version.

Reply:
I just did a commit that reworked the ParseResult to include the $unparsed logic and updated all the other parsers. The idea is to fill it only if the invType did not match from the DB. My main use case is I have corp members playing in french and the item names do not match in the database because they need to put the important names in english in game.